add Community helps to getting started page and add 2 solid apps to apps page (#799)
solid/solidproject.org: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md

## Developing your first app
Once you are set up with your own Pod and WebID,
you're ready to start [developing your own app](/developers/tutorials/first-app).

> ## Community helps

Reviewer: ```suggestion
## Community assistance
```

Author: fixed

> Check some of the applications. Some interesting approaches

Reviewer: ```suggestion
Check some of the applications. Some interesting approaches follow:
```

Author: appliyed

> - PodOS https://github.com/pod-os/PodOS (Web Components)
> - solid-uix https://github.com/SolidOS/solid-uix (Native Web Components)

Reviewer: ```suggestion
- **PodOS** (Web Components) https://github.com/pod-os/PodOS
- **solid-uix** (Native Web Components) https://github.com/SolidOS/solid-uix
```

Author: appliyed

Solid With web frameworks.

- ember-solid: [Video Tutorial](https://nextcloud.hellofuture.be/s/nYYNwGceFSNotJY)  which should be start-to-finish Solid Ember App (Aad Versteden).
- take a look at [todo-ldo](https://github.com/megoth/todo-ldo) a [solid/nextjs/ts] app (Arne Hassel)
- and also this one [solid-react-app-tutorial](https://virginiabalseiro.com/blog/solid-react-app-tutorial)

Cou can also checkout the solid apps at:

> - [Solid Apps](https://solidproject.org/apps)

Author: appliyed

## Developing your first app

Once you are set up with your own Pod and WebID,

> you're ready to develop your own app. Various Solid client libraries may provide a beginning app tutorial; for example, the [Inrupt JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).

Reviewer: ```suggestion
you're ready to develop your own app. Various Solid client libraries may provide a  tutorial for beginners; for example, Inrupt offers the [JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).
```

## Community assistance

> Check some of the applications. Some interesting approaches follow:

Reviewer: ```suggestion
Check out some of the existing applications. Some interesting approaches follow:
```

- **PodOS** (Web Components) https://github.com/pod-os/PodOS
- **solid-uix** (Native Web Components) https://github.com/SolidOS/solid-uix

Solid With web frameworks:

- **ember-solid**: [Video Tutorial](https://nextcloud.hellofuture.be/s/nYYNwGceFSNotJY)  which should be start-to-finish Solid Ember App (Aad Versteden)
- **[todo-ldo](https://github.com/megoth/todo-ldo)** a [solid/nextjs/ts] app (Arne Hassel)
- **[solid-react-app-tutorial](https://virginiabalseiro.com/blog/solid-react-app-tutorial)**

You can also look into the Solid apps at:
- [Solid Apps](https://solidproject.org/apps)

Youtube videos:

> - [How we're doing Solid - In less than 5 minutes](https://www.youtube.com/watch?v=CqI15zs1NHw)

Reviewer: ```suggestion
- [How we're doing Solid — in fewer than 5 minutes](https://www.youtube.com/watch?v=CqI15zs1NHw)
```

## Developing your first app

Once you are set up with your own Pod and WebID,

> you're ready to develop your own app. Various Solid client libraries may provide a  tutorial for beginners; for example, Inrupt offers the [JS Client Library tutorial](https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/getting-started/).

Author: applied

## Community assistance

Check out some of the existing applications. Some interesting approaches follow:

- **PodOS** (Web Components) https://github.com/pod-os/PodOS
- **solid-uix** (Native Web Components) https://github.com/SolidOS/solid-uix

> Solid With web frameworks:

Author: applied

> - **ember-solid**: [Video Tutorial](https://nextcloud.hellofuture.be/s/nYYNwGceFSNotJY)  which should be start-to-finish Solid Ember App (Aad Versteden)

Author: applied

- **[todo-ldo](https://github.com/megoth/todo-ldo)** a [solid/nextjs/ts] app (Arne Hassel)

> - **[solid-react-app-tutorial](https://virginiabalseiro.com/blog/solid-react-app-tutorial)**

Author: applied
